scripts/system/create/entityProperties/html/js/entityProperties.js

```diff
@@ -3954,44 +4007,36 @@ function handleEntitySelectionUpdate(selections, isPropertiesToolUpdate) {
 
 function loaded() {
     openEventBridge(function() {
-        let elPropertiesList = document.getElementById("properties-list");
+        let elPropertiesList = document.getElementById("properties-pages");
+        let tabs = "";
```

Comment:
Why not create a element for the tabs?

Reply:
I went initially with what I was comfortable with. 
Now that the solution work, I will make it compliant to this.

Reply:
Code changed accordingly.